rats-tls/samples/rats-tls-client/client.c

```diff
@@ -143,7 +143,12 @@ int rats_tls_client_startup(rats_tls_log_level_t log_level, char *attester_type,
 		goto err;
 	}
 
-	const char *msg = "Hello and welcome to RATS-TLS!\n";
+	const char *msg;
+	if (verdictd)
+		msg = "{ \"command\": \"echo\", \"data\": \"Hello and welcome to RATS-TLS!\" }";
```

Comment:
Can wen remove \"data\" and just keep the msg as "command: echo: XXXX"? That will be more brief.

Reply:
Thank you for your comments. 
The communication between `rats-tls-client` and `verdictd` is based on EAA protocol.
The echo command in EAA protocol is defined as followed:
```JSON
{
    "command": "echo"
    "data": "xxxxxxx"
}
```